### DISCUSSION TITLE
Feature request: translatable checkbox formfield

### DISCUSSION THREAD

**Discussion by jeepey** [area17/twill#1003] [Ideas]
Hi,

While looking at the code it seems like localization is not implemented on the checkbox fieldtype. 
Might be an odd use-case but we would like to use it. I tried to make a custom formfield type but somehow the corresponding view cannot be resolved by the code (I placed it in 'admin/partials/form/').

Thanks!

**Comment by ifox** [comment]
Hi @jeepey,

That should be possible, I can see why you would need it. Would you mind sharing your implementation?

You should able to create a custom form field partial at `resources/views/admin/partials/form/_<form_field_name>.blade.php`. Maybe you forgot the underscore?
